controllers/apply/awards-for-all/fields/project-budget.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+const get = require('lodash/fp/get');
+const { oneLine } = require('common-tags');
+
+const Joi = require('../../lib/joi-extensions');
+
+const {
+    MAX_BUDGET_TOTAL_GBP,
+    MIN_BUDGET_TOTAL_GBP,
+    FREE_TEXT_MAXLENGTH
+} = require('../constants');
+
+module.exports = function(locale) {
+    const localise = get(locale);
+
+    const ROW_LIMIT = 10;
+
+    return {
+        name: 'projectBudget',
+        label: localise({
+            en: 'List the costs you would like us to fund',
+            cy: 'Rhestrwch y costau hoffech i ni eu hariannu'
+        }),
+        explanation: localise({
+            en: `<p>
+                You should use budget headings, rather than a detailed list
+                of items. For example, if you're applying for pens, pencils,
+                paper and envelopes, using 'office supplies' is fine.
+                Please enter whole numbers only.
+            </p>
+            <p>Please note you can only have a maximum of 10 rows.</p>`,
```

Comment:
This isn't part of this PR really but should the "10" here use the `ROW_LIMIT` above?